### PR TITLE
Source Airtable: skip missing streams

### LINKED ISSUE #25547: Source: Airtable - stream missing during sync
## What
Users are reporting that a stream is missing during sync and cannot be found in the available streams. This issue is causing syncing problems and affecting the overall functionality of the application.

### Workspaces:

- https://cloud.airbyte.com/workspaces/3a00c862-22cb-48eb-998b-961c5e322959/connections/7fd26760-a3d2-4146-b93b-38db84946b30/status#1889068::2

- https://cloud.airbyte.com/workspaces/904e24b5-5be5-4da3-b99d-3ca0f43deab5/connections/8c49f0b8-4ca0-4990-a2f0-0167f1e5906f/status#1974135::2

### PR DESCRIPTION
**Pull request by arsenlosenko**
## What
Resolving issue described here:
https://github.com/airbytehq/airbyte/issues/25547

## How
Considering that the issue with missing streams most likely occurs if table on which stream is based on may be renamed/removed in Airtable I have added a check to take this into account. In case this happens, user will be presented with a warning, describing the issue and pointing to docs, and the stream in question will be removed from catalog till the data is reset.

## 🚨 User Impact 🚨
This is a patch change, tweaking the way `read` performs if the stream is missing

### CODE REVIEW

**Review by davydov-d — APPROVED** [review]

**Review by bnchrch — APPROVED** [review]
LGTM! 👍

File: docs/integrations/sources/airtable.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ When using OAuth, you may see a `400` or `401` error causing a failed sync. You
 5. Click **Set up source**.
 <!-- /env:oss -->
 
+### Note on changed table names
```
Comment (sherifnada) [inline docs/integrations/sources/airtable.md:47]:
```suggestion
### Note on changed table names & deleted tables
```

**Review by sherifnada — APPROVED** [review]

**Pull request merged**